refactor: updates to latest Gravity release

james-nash opened the pull request:
Bumps to the latest Gravity NEXT pre-release, which introduces new and updated multi-column layout
classes. This change also updates some of the website templates' mark-up to use those new classes
correctly.

dw-buildit commented on package.json:
```diff
@@ -56,9 +56,9 @@
   "dependencies": {
+    "@buildit/gravity-ui-web": "^3.0.0-alpha.6",
     "@buildit/job-listings": "^1.1.0",
-    "@buildit/gravity-ui-web": "next",
```
shouldn't this remain as `next`? Just the `package-lock.json` will change to a specific alpha.6 version?

james-nash replied:
Ah yes, you're right. Will fix that.

james-nash commented:
@dw-buildit I fixed the `package.json` issue you spotted.

I also noticed a layout problem caused by our new column layout mixin: The story list's negative margins make it wider than the viewport (even if the positioning makes it _look_ like it doesn't). So, on smaller viewports this caused the page to be wider than the viewport and you got horrible horizontal scrolling.

It needs to be fixed upstream in Gravity but might be non-trivial, since we'll need to find a different way of doing the column layouts. I'll raise an issue for this in a moment.

But, to not hold up website stuff in the short term, I added a small band-aid in the website's SASS (`overflow-x: hidden;` on the `.grav-o-container`), to work around this.

dw-buildit commented on src/sass/style.scss:
```diff
@@ -44,3 +44,9 @@
+// column layout styling that causes the page to
+// overflow the viewport horizontally.
+.grav-o-container {
+  overflow-x: hidden;
```
This isn't lovely 😆 

Not going to block the merge for it though

james-nash replied:
I know - far from it.

Thanks for approving though!
